Headlines/src/ViewAnimator/ViewAnimator.swift

```diff
@@ -0,0 +1,113 @@
+                       usingSpringWithDamping: dampingRatio,
+                       initialSpringVelocity: velocity,
+                       options: options,
+                       animations: { [weak self] in
+            self?.transform = reversed ? transformTo : transformFrom
+            self?.alpha = finalAlpha
+        }) { _ in
+            completion?()
+        }
+    }
+    
+    /// Performs the animation.
+    ///
+    /// - Parameters:
+    ///   - animations: Array of Animations to perform on the animation block.
+    ///   - reversed: Initial state of the animation. Reversed will start from its original position.
+    ///   - initialAlpha: Initial alpha of the view prior to the animation.
+    ///   - finalAlpha: View's alpha after the animation.
+    ///   - delay: Time Delay before the animation.
+    ///   - animationInterval: Interval between the animations of each view.
+    ///   - duration: TimeInterval the animation takes to complete.
+    ///   - dampingRatio: The damping ratio for the spring animation.
+    ///   - velocity: The initial spring velocity.
+    ///   - completion: CompletionBlock after the animation finishes.
+    static func animate(views: [UIView],
+                        animations: [Animation],
+                        reversed: Bool = false,
+                        initialAlpha: CGFloat = 0.0,
+                        finalAlpha: CGFloat = 1.0,
+                        delay: Double = 0,
+                        animationInterval: TimeInterval = 0.05,
+                        duration: TimeInterval = ViewAnimatorConfig.duration,
+                        usingSpringWithDamping dampingRatio: CGFloat = ViewAnimatorConfig.springDampingRatio,
+                        initialSpringVelocity velocity: CGFloat = ViewAnimatorConfig.initialSpringVelocity,
+                        options: UIView.AnimationOptions = [],
+                        completion: (() -> Void)? = nil) {
+
+        guard views.count > 0 else {
+            completion?()
+            return
+        }
+        
+        views.forEach { $0.alpha = initialAlpha }
+        let dispatchGroup = DispatchGroup()
+        for _ in 1...views.count { dispatchGroup.enter() }
+        DispatchQueue.main.asyncAfter(deadline: .now() + delay) {
+            for (index, view) in views.enumerated() {
+                view.alpha = initialAlpha
+                view.animate(animations: animations,
+                             reversed: reversed,
+                             initialAlpha: initialAlpha,
+                             finalAlpha: finalAlpha,
+                             delay: Double(index) * animationInterval,
+                             duration: duration,
+                             usingSpringWithDamping: dampingRatio,
+                             initialSpringVelocity: velocity,
+                             options: options,
+                             completion: { dispatchGroup.leave() })
+            }
+        }
+        
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/a5cc79d8b0bc8bc433e47d6a58e00624ade77840/Headlines/src/ViewAnimator/ViewAnimator.swift#L108"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>ViewAnimator.swift:108</code></td>
    </tr>
  </tbody>
</table>